zulip_bots/zulip_bots/bots/converter/converter.py

```diff
@@ -8,9 +8,11 @@
 from math import log10, floor
 
 import re
-from zulip_bots.bots.converter import utils
+from zulip_bots.zulip_bots.bots.converter import utils
```

Comment:
This import is wrong, the previous import was correct! :) So I would suggest changing this back to what it was. :)

Reply:
This is also why Travis is failing! :)